app/coffeescripts/models/Assignment.coffee

```diff
@@ -356,6 +372,8 @@ define [
         data.assignment_overrides = new AssignmentOverrideCollection overrides
       if (turnitin_settings = data.turnitin_settings)?
         data.turnitin_settings = new TurnitinSettings turnitin_settings
+      if (vericite_settings = data.vericite_settings)?
+        data.vericite_settings = new VeriCiteSettings vericite_settings
```

Comment:
"vericite_settings" is a logical function that uses the "turnitin_settings" database field. This allows VeriCite and Turnitin to have different advanced settings while still using the same database field.